### PR TITLE
Mobile responsive

### PR DESCRIPTION
**Pull request by Afsheen96**
In this project, you will set up a "Portfolio" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
General requirements
Once you finish your project, double-check to make sure your repo is ready:
Make sure that there are no linter errors.
Make sure that you used the correct (GitHub flow).
Make sure that you documented your work.
Requirements
Remember to document your work in a professional way.
Create a new GitHub repo.
Create a new branch according to the GitHub flow rules.
Add a .gitignore file.
Add node_modules to the gitignore file.
Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
Add a descriptive README file to your project - please use this template but remember to customize it to your project.
Add 3 files (each one in a separate commit):
A .html file that includes a header and section like the figma template
A .css file that includes  styles for your HTML elements according to figma template.
A test. MD file with the text "This file should be ignored by git".
Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
Open a pull request.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <img src="resources/images/Logo.png" alt="my logo">
```
Comment (usorfaitheloho) [inline index.html:14]: Kindly include the `logo` in an `a` tag for semantics, convention and a good user experience 😄

File: style.css [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+article {
+  display: flex;
+  min-height: 113vh;
+  flex-direction: column;
+  background-color: white;
+  border-bottom-left-radius: 15%;
+  background-image: url("./resources/images/header-shapes\ mobilebackground-Main.png");
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-position: 99% -23%;
```
Comment (usorfaitheloho) [inline style.css]: you did well adding the background image, kindly use the right spacing between the nav and the background image for your project as it is seen in the Figma template given 😄 

Figma design :
image

Your design :
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+              I’m Afsheen Kiran. <br />
+              Glad to see you!
+            </h1>
+          </div>
+          <div class="article item2">
+            <p class="presentation">
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hesitate to contact me.
```
Comment (usorfaitheloho) [inline index.html:33]: Please kindly use the right font color for your paragraph similar to what we have in the design which is `#344563`

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @Afsheen96  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Good design ✔️
> * Followed HTML and CSS best practices ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @Afsheen96 ,

Great job so far! 

May I highlight the following: 
- [x] Your app is working perfectly👏  ✔️ 
- [x] You implemented all the required changes by the previous TSE✔️ 



## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**